kentikapi/internal/api_resources/custom_dimensions_test.go

```diff
@@ -94,95 +98,238 @@ func TestUpdateCustomDimension(t *testing.T) {
 }
 
 func TestGetCustomDimension(t *testing.T) {
-	// arrange
-	getResponsePayload := `
-	{
-		"customDimension": {
-			"id": 42,
-			"name": "c_testapi_dimension_1",
-			"display_name": "dimension_display_name",
-			"type": "string",
-			"company_id": "74333",
-			"populators": [
-				{
-					"id": 1510871096,
-					"dimension_id": 24001,
-					"value": "testapi-dimension-value-1",
-					"direction": "DST",
-					"device_name": "128.0.0.100,device1",
-					"interface_name": "interface1,interface2",
-					"addr": "128.0.0.1/32,128.0.0.2/32",
-					"addr_count": 2,
-					"port": "1001,1002",
-					"tcp_flags": "160",
-					"protocol": "6,17",
-					"asn": "101,102",
-					"nexthop_asn": "201,202",
-					"nexthop": "128.0.200.1/32,128.0.200.2/32",
-					"bgp_aspath": "3001,3002",
-					"bgp_community": "401:499,501:599",
-					"user": "144319",
-					"created_date": "2020-12-15T08:32:19.503788Z",
-					"updated_date": "2020-12-15T08:32:19.503788Z",
+	tests := []struct {
```

Comment:
Uncle Bob would frown at the sight of function this long :D

Reply:
Maybe, but most of the function is just test data. I could split it to functions, but I doubt that it would increase visibility :)